src/Symfony/Component/Console/Input/Input.php

```diff
@@ -134,12 +118,7 @@ public function getArgument($name)
     }
 
     /**
-     * Sets an argument value by name.
-     *
-     * @param string $name  The argument name
-     * @param string $value The argument value
-     *
-     * @throws \InvalidArgumentException When argument given doesn't exist
```

Comment:
In fact the exception class is `Console\Exception\InvalidArgumentException`, same for other.
Do we add a use statement in this case? I used the parent exception so it stays correct though less precise.